pkgs/by-name/md/mdbook/package.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, stdenv, fetchFromGitHub, nix, rustPlatform, CoreServices, installShellFiles }:
-
-rustPlatform.buildRustPackage rec {
+{ lib, stdenv, fetchFromGitHub, nix, rustPlatform, CoreServices, installShellFiles }: let
```

Comment:
I'd recommend formatting the file with `nixfmt-rfc-style`.

Reply:
I'll go ahead and do that then!